site/instrument.ts

```diff
@@ -0,0 +1,35 @@
+import * as Sentry from "@sentry/react"
+import { isInIFrame } from "@ourworldindata/utils"
+import { COMMIT_SHA, ENV, SENTRY_DSN } from "../settings/clientSettings.js"
+import { getPreferenceValue, PreferenceType } from "./cookiePreferences.js"
+
+const analyticsConsent = getPreferenceValue(PreferenceType.Analytics)
```

Comment:
Same as before, but this means we'll never be able to log sessions from new users on their first load of a page?

i.e.
1. Load homepage
2. Sentry initializes without session tracking
3. Click "Accept cookies"

I guess it wouldn't make sense to begin recording a session midway (though it would be kind of funny to see them "turning the camera on" by clicking the "Accept cookies" button.

Oh well. Maybe check with Bobbie if he thinks these new user sessions would be especially useful, but my sense is that this is fine.